build/cite/wfs20/compose.override.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  geoserver:
+    image: ogccite/geoserver:wfs11
```

Comment:
should this not be `ogccite/geoserver:wfs20`?

looking at https://github.com/geoserver/geoserver/blob/352c0a8d7121107a4e1d5cab18b08c4169bacae6/.github/workflows/cite.yml#L58 
and 
https://github.com/geoserver/geoserver/blob/352c0a8d7121107a4e1d5cab18b08c4169bacae6/.github/workflows/cite.yml#L83C43-L85
that image seems to be build as part of the workflow